core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -354,3 +354,8 @@ object SupportedMLFuncsName {
 }
 
 case class GpuEventLogException(message: String) extends Exception(message)
+
+object GpuTypes {
```

Comment:
Nit: If we introduce GpuTypes object, then it would make more sense that it is a "case class" lookup table that will include all the information we need. For example: the name, and the memory size .etc.
Then we won't need a map like `DEF_WORKER_GPU_MEMORY_MB`.

Reply:
Fixed, thanks